compatibility/bazel_tools/daml_ledger/Main.hs

```diff
@@ -91,9 +91,13 @@ withTools tests = do
           }
   tests tools
 
+-- | This is the version of daml-helper.
 newtype SdkVersion = SdkVersion String
+  deriving Eq
 instance IsOption SdkVersion where
-  defaultValue = SdkVersion (error "SDK version has to be set explicitly using --sdk-version")
+  defaultValue = SdkVersion "0.0.0"
+  -- Tasty seems to force the value somewhere so we cannot just set this
```

Comment:
I’m not actually sure what is going on here. I tried changing `SdkVersion` to `data` but it still crashes. I did however, verify that we set the version correctly so the default is really unused.